cli/write.go

```
@@ -1019,27 +1026,66 @@ func showProvides(c *cli.Context) (map[string]string, error) {
 
 	cmd := exec.Command("ssh", args...)
 
-	stdout, err := cmd.CombinedOutput()
-
+	// Simply connect stdin/stderr
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+	stdout, err := cmd.StdoutPipe()
 	if err != nil {
+		return nil, errors.New("Error redirecting stdout on exec")
+	}
+
+	// Disable tty echo before starting
+	term, err := util.DisableEcho(int(os.Stdin.Fd()))
+	if err == nil {
+		sigChan = make(chan os.Signal, 1)
+		errChan = make(chan error, 1)
+		// Make sure that echo is enabled if the process gets
+		// interrupted
+		signal.Notify(sigChan)
+		go util.EchoSigHandler(ctx, sigChan, errChan, term)
+	} else if err != syscall.ENOTTY {
+		return nil, err
+	}
```

Are we intentionally ignoring all errors except ENOTTY?